spec/controllers/admin/customers_controller_spec.rb

```diff
@@ -164,19 +164,27 @@ def expect_locals
       customer = create(:customer)
 
       expect do
-        delete :destroy, {id: customer.to_param}
+        delete :destroy, kwarg_params(id: customer.to_param)
       end.to change(Customer, :count).by(-1)
     end
 
     it "redirects to the customers list" do
       customer = create(:customer)
 
-      delete :destroy, id: customer.to_param
+      delete :destroy, kwarg_params(id: customer.to_param)
 
       expect(response).to redirect_to(admin_customers_url)
     end
   end
 
+  def kwarg_params(args)
+    if Rails::VERSION::MAJOR >= 5
+      {params: args}
+    else
+      args
+    end
```

Comment:
Yep, that's another solution. I would opt for the ternary mainly because I think the purpose of the method is to format the params if its greater than or equal to Rails 4... If we had to use guard clauses, I think a more optimal outcome would be:

```ruby
return args if Rails::VERSION::MAJOR <= 4
{ params: args }
```

🚲 🏠